rocketpy/Rocket.py

```diff
@@ -564,11 +564,16 @@ def addFins(
         cantAngle : int, float, optional
             Fins cant angle with respect to the rocket centerline. Must
             be given in degrees.
-        airfoil : string
-            Fin's lift curve. It must be a .csv file. The .csv file shall
-            contain no headers and the first column must specify time in
-            seconds, while the second column specifies lift coefficient. Lift
-            coefficient is dimensionaless.
+        airfoil : tuple, null
+            Airfoil's lift curve. If Null, fins will be considered
+            planar. If tuple, fins will be considered airfoiled.
+            The tuple must contain as its first item a .csv or .txt
+            file with the aifoil's lift coefficient by angle of attack
+            curve. The file must contain no headers and the first column
+            must specify the angle of attack, while the second column
+            must specify the lift coefficient. The tuple must contain as
+            its second item the unit of the angle of attack, accepting
+            either "radians" or "degrees".
```

Comment:
Please update the argument description to include the fact that the first item can also be a list or a callable (function), or even a Function actually (no need to mention this, since a Function is callable). See the description of the source argument of the `Function.__init__` method for more details. This is important to handle #55.

Also, make sure to flag this parameter as optional: `airfoil : tuple, optional`